tests/global_communication/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ set(TARGETS global_communication.exe)
 
 foreach(target ${TARGETS})
     target_link_libraries(${target} LINK_PUBLIC nestmc gtest)
+    target_link_libraries(${target} LINK_PUBLIC ${EXTERNAL_LIBRARIES})
 
     if(WITH_TBB)
```

Comment:
The TBB library is in `EXTERNAL_LIBRARIES`, so you can remove the `if(WITH_TBB)` block once you have added the above link line.